dev/tests/integration/testsuite/Magento/Framework/App/Utility/FilesTest.php

```diff
@@ -159,4 +159,16 @@ public function testReadLists()
         }
         $this->assertSame($expected, $actual);
     }
+
+    /**
+     * Tests if underscores are allowed in extended static files in theme.
+     *
+     * @magentoComponentsDir Magento/Framework/App/Utility/_files/design
+     * @magentoAppIsolation enabled
+     * @magentoDbIsolation enabled
+     */
+    public function testAllowUnderscoreInExtendedFiles()
+    {
+        $this->model->getStaticPreProcessingFiles('*.less');
```

Comment:
I believe we should make sure that we got the correct list of files. What do you think?

Reply:
@ihor-sviziev updated.